apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -224,7 +224,7 @@ import data_converter from "../../mixins/data_converter";
 import DataMutators, { escapeHtml } from '../../mixins/data_mutators'
 import { FkLinkMixin } from '@/mixins/fk_click'
 import Statusbar from '../common/StatusBar.vue'
-import RowFilterBuilder from './RowFilterBuilder.vue'
+import RowFilterBuilder, { normalizeFilters } from './RowFilterBuilder.vue'
```

Comment:
You should move this function somewhere else, Vue files should really only contain the vue component. Maybe utils?

Reply:
done :)